src/screens/Survey/index.tsx

```diff
@@ -1,10 +1,49 @@
 import React from 'react';
-import { useParams } from 'react-router-dom';
 
-const SurveyScreen = (): JSX.Element => {
-  const { id } = useParams();
+import { ReactComponent as ArrowBack } from 'assets/images/icons/arrow-back.svg';
+import BackgroundImage from 'components/BackgroundImage';
+import ElevatedButton from 'components/ElevatedButton';
+
+export const surveyScreenTestIds = {
+  backButton: 'survey__back-button',
+  coverImage: 'survey__cover-image',
+  title: 'survey__title-text',
+  description: 'survey__description-text',
+  startSurveyButton: 'survey__start-survey-button',
+};
 
-  return <div>Dashboard detail {id}</div>;
+const SurveyScreen = (): JSX.Element => {
+  return (
+    <BackgroundImage backgroundUrl="https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_">
```

Comment:
It's better to update the component's name, `BackgroundImage` sounds like a `img` tag, not a component has children

Reply:
@nvminhtue I have a name in mind, how about calling this widget `MainView` since it's always a parent widget? 😃